PWGPP/CMakeLists.txt

```diff
@@ -332,6 +334,10 @@ install(FILES HMPID/AddTaskHmpidQA.C DESTINATION PWGPP/HMPID)
 install(FILES HMPID/macros/copyHMPQAfiles.C
  	      HMPID/macros/makeHMPQA.C DESTINATION PWGPP/HMPID/macros)
 install(FILES T0/AddTaskT0QA.C
+        T0/spectrum/AddAnalysisTaskSpectrumT0.C
+        T0/spectrum/runListRCT2016_pp_tzero.root
```

Comment:
These are small files, I guess it is fine to have them in AliPhysics. @shahor02, what do you think?